folium/folium.py

```diff
@@ -404,9 +413,12 @@ def div_markers(self, locations=None, popups=None, marker_size=10, popup_width=3
                                        'icon': "{'icon':"+icon_name+"}"
                                        })
 
-            popup_out = self._popup_render(popup=popup, mk_name='div_marker_{0}_'.format(call_cnt),
```

Comment:
Why do you fix this line like this instead of something like

    mk_out = 'div_marker_{0}_'.format(call_cnt)
    popup_out = self._popup_render(popup=popup, mk_name=mk_out)

?

I don't know pep8 very well, but it seems to me that it would be more understandable.